lib/fs.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const fs = require('fs');
+
+const mkdir = path => {
+  if (!fs.existsSync(path))
+    fs.mkdirSync(path, err => {
+      if (err) throw err;
+    });
+};
+
+const mkdirp = path => {
+  const version = process.versions.node.split('.').map(el => parseInt(el));
+  if (version[0] < 10 || (version[0] === 10 && version[1] <= 11)) {
+    let firstIndex = 0;
+    for (let i = 1; i < path.split('/').length; i++) {
```

Comment:
Shouldn't we also support Windows platform?

Reply:
How should I do that? Could you explain in full?

Reply:
You can use `path.dirname` to get parent folder and `path.join` to get child folder.